cmd/serverless/main.go

```diff
@@ -15,9 +15,11 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/DataDog/datadog-agent/pkg/logs/message"
```

Comment:
Import order seems wrong, these need to be grouped with other `"github.com/DataDog/datadog-agent/pkg/..."` imports together